### ISSUE TITLE
Option to inject scripts instead of reload

### ISSUE THREAD

**Issue by iclanzan** [BrowserSync/browser-sync#675]
I would love to see an option to inject JS files instead of refreshing the page.

**Comment by shakyShane** [comment]
This would be completely dependant on how your JS is written & not something that could be universally applied (because of browser state). 

Would be interesting to see a plugin based on this idea though

**Comment by iclanzan** [comment]
Yes, you are right. Could you point me in the right direction with writing a plugin that does this? Can’t find any documentation about writing plugins for `browser-sync`.

**Comment by shakyShane** [comment]
Sorry the docs re: plugins are not available yet as I'm not 100% with internals/api/hooks etc. I hope to change this very soon though.
